adagrams/game.py

```diff
@@ -1,13 +1,152 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+    1: ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+    2: ["D", "G"],
+    3: ["B", "C", "M", "P"],
+    4: ["F", "H", "V", "W", "Y"],
+    5: ["K"],
+    8: ["J","X"],
+    10: ["Q","Z"]  
+}
+
+def get_draw_pool():
+    draw_pool = []
+
+    for letter, num in LETTER_POOL.items():
+        while num > 0:
+            draw_pool.append(letter)
+            num -= 1
+            # print(num)
+    return draw_pool
```

Comment:
Love that you made a helper function here! These leads to smaller batches of code which leads a more streamlined debugging process.

Something to note here is that the `while` loop you wrote is just being used to run your indented logic however many times the current letter is able to appear in the pool. Since you basically using it as a `for range` loop, we could just use a `for range` loop.
```suggestion
def get_draw_pool():
    draw_pool = []

    for letter, num in LETTER_POOL.items():
        for _ in range(num):
            draw_pool.append(letter)
            num -= 1
            
    return draw_pool
```